### PR TITLE
changes for 2.35

### PR DESCRIPTION
**Pull request by catalinaadam**

### CODE REVIEW

File: docs/get-started/backup/mongo-prerequisites.md [hunk 1]
```diff
@@ -7,9 +7,14 @@ Before creating MongoDB backups, make sure to:
...
 5. Check that [Percona Backup for MongoDB](https://docs.percona.com/percona-backup-mongodb/index.html) (PBM) is installed and `pbm-agent` is running on all MongoDB nodes in the replica set. PMM 2.32 and later require PBM 2.0.1 or newer.
-6. Check that MongoDB is a member of a replica set.
-7. Check that you set the [required permissions for creating and restoring MongoDB backups](../../setting-up/client/mongodb.md#create-pmm-account-and-set-permissions).
-8. Verify the [MongoDB supported configurations and limitations](../../get-started/backup/mongodb_limitations.md).
+6. Check that your MongoDB Services are managed as a cluster. Go to **PMM Inventory > Services** page and make sure that all the services in the table specify a cluster name in the **Other Details** column. Services that do not specify a cluster name should be removed and re-added using the following command:
```
Comment (PavelKhripkov) [inline docs/get-started/backup/mongo-prerequisites.md:10]:
```suggestion
6. Check that your MongoDB Services are managed as clusters in PMM. Go to **PMM Inventory > Services** page and make sure that all the services in the table specify a cluster name in the **Other Details** column. Services that do not specify a cluster name should be removed and re-added using the following command:
```

File: docs/get-started/backup/mongo-prerequisites.md [hunk 2]
```diff
@@ -7,9 +7,14 @@ Before creating MongoDB backups, make sure to:
...
+
+       <pre>pmm-admin add mongodb
+       username=pmm_mongodb --password=password 
+       query-source=profiler <mark>--cluster=mycluster</mark></code></pre>
```
Comment (PavelKhripkov) [inline docs/get-started/backup/mongo-prerequisites.md:14]: It's hard to find command in the article text. Can we highlight it with frame or something?

File: docs/get-started/backup/mongo-prerequisites.md [hunk 3]
```diff
@@ -7,9 +7,14 @@ Before creating MongoDB backups, make sure to:
...
+7. Check that MongoDB is a member of a replica set.
```
Comment (PavelKhripkov) [inline docs/get-started/backup/mongo-prerequisites.md:15]:
```suggestion
7. Check that MongoDB nodes are members of replica set.
```

File: docs/get-started/backup/restore_MongoDB_backups.md [hunk 4]
```diff
@@ -105,9 +105,9 @@ This opens the **Restore from backup** dialog, with the **Same service** option
 
           `pbm restore --time="2022-11-23T19:40:26`
         
-      For more information, see [Point-in-time Recovery topic in the PBM documentation](https://docs.percona.com/percona-backup-mongodb/usage/point-in-time-recovery.html).
+      For more information, see the[Point-in-time Recovery topic in the PBM documentation](https://docs.percona.com/percona-backup-mongodb/usage/point-in-time-recovery.html).
```
Comment (PavelKhripkov) [inline docs/get-started/backup/restore_MongoDB_backups.md:108]:
```suggestion
      For more information, see the [Point-in-time Recovery topic in the PBM documentation](https://docs.percona.com/percona-backup-mongodb/usage/point-in-time-recovery.html).
```

**Review by PavelKhripkov — CHANGES_REQUESTED** [review]

File: docs/get-started/backup/mongo-prerequisites.md (hunk 2 above)
Comment (catalinaadam) [reply docs/get-started/backup/mongo-prerequisites.md:14]: It renders differently on the doc website. check out the [preview ](https://two-34-0-pr-994.onrender.com/get-started/backup/mongo-prerequisites.html)

**Pull request merged**